plugins/teststeps/teststeps_test.go

```diff
@@ -217,12 +225,12 @@ func TestForEachTargetTenTargetsParallelism(t *testing.T) {
 	logging.Debug()
 	sleepTime := time.Second
 	d := newData()
-	fn := func(cancel, pause <-chan struct{}, tgt *target.Target) error {
+	fn := func(ctx statectx.Context, tgt *target.Target) error {
 		log.Printf("Handling target %+v", tgt)
 		select {
-		case <-cancel:
+		case <-ctx.Done():
 			log.Printf("target %+v caneled", tgt)
```

Comment:
Since we are here already (also in more comments in this file)
```suggestion
			log.Printf("target %+v cannelled", tgt)
```